docs/contribution-guides/Starting-contribution.md

```diff
@@ -150,7 +150,7 @@ A **P**ull **R**equest is the only way to get your changes into the official ver
 
 You will see a page where you can name your PR and fill a little form with the relevant information. Your PR will be tested and reviewed and then it could be approved to be merged into the game!
 
-Please ensure you are familiar with our [development standards](https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide) before submitting your PR.
+
```

Comment:
Ummm are you sure you inteded to remove this? 😅

Reply:
Most of the files I committed were because I'm a moron and forgot to uncheck several files Unity edited for no reason. This was one of them.